src/UI/Component/Link/Factory.php

```diff
@@ -30,5 +34,47 @@ interface Factory {
 	 * @param	string		$action
 	 * @return  \ILIAS\UI\Component\Link\Standard
 	 */
-	public function standard($label, $action);
+	public function standard(string $label, string $action): Standard;
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The Bulky Link is highly obtrusive. It combines the recognisability
+	 *     of a graphical element with an explicit textual label on an unusually
+	 *     sized button-like area.
+	 *
+	 *   composition: >
+	 *     The Bulky Link is built as a Link around an icon or glyph and a (small) text.
+	 *
+	 *   rivals:
+	 *     Bulky Button: >
+	 *       Although visually very much alike, Bulky Buttons rather trigger a Signal
+	 *       and execute JavaScript while the Bulky Link opens a URL.
+	 *       Use Buttons to act upon other elements and Links to change the page.
+	 *       Bulky Links are not stateful.
+	 *
+	 * context:
+	 *   - Slate
+	 *   - Drilldown Menu
+	 *
+	 * rules:
+	 *   wording:
+	 *     1: The icon/glyph and the text of the Bulky Link MUST be corresponding.
+	 *   style:
+	 *     1: >
+	 *       Bulky Links MUST occupy as much space as their container leaves them.
+	 *   responsiveness:
+	 *     1: >
+	 *        On screens larger than small size, Bulky Links MUST contain an icon or glyph plus text.
+	 *     2: >
+	 *        On small-sized screens, Bulky Links SHOULD contain only an icon or glyph.
+	 *
+	 * ---
+	 * @param	\ILIAS\UI\Component\Symbol\Symbol		$symbol
+	 * @param	string		$label
+	 * @param	string		$action
+	 * @return  \ILIAS\UI\Component\Link\Bulky
+	 */
+	public function bulky(Symbol $symbol, string $label, string $action): Bulky;
```

Comment:
Lets use `Data\URI` here instead of the string. Let's rename `$action` to `$target`. A link has a target, right?.